projects/aca-content/src/lib/components/search/search-save/list/smart-list/saved-searches-smart-list.component.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, inject, OnDestroy, OnInit, ViewEncapsulation } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TranslateModule } from '@ngx-translate/core';
+import { SavedSearchesListUiComponent } from '../ui-list/saved-searches-list.ui-component';
+import { AppService, PageComponent, PageLayoutComponent } from '@alfresco/aca-shared';
+import { MaterialModule, SavedSearchesService } from '@alfresco/adf-content-services';
+import { EmptyContentComponent } from '@alfresco/adf-core';
+
+@Component({
+  selector: 'aca-saved-searches-smart-list',
+  standalone: true,
+  imports: [CommonModule, TranslateModule, SavedSearchesListUiComponent, PageLayoutComponent, EmptyContentComponent, MaterialModule],
```

Comment:
MaterialModule is deprecated: 
<img width="591" alt="image" src="https://github.com/user-attachments/assets/14628b21-bfb8-4f24-ac2f-aa763d1f9763">

Reply:
Removed